auraed/src/cells/cell_service/cell_service.rs

```diff
@@ -187,8 +194,21 @@ impl CellService {
             .expect("pid")
             .as_raw();
 
-        // TODO: either tell the [ObserveService] about this executable's log channels, or
-        // provide a way for the observe service to extract the log channels from here.
+        // Ensure the observe service is aware of the executable's logs.
+        self.observe_service
+            .register_sub_process_channel(
```

Comment:
If these register calls error, the client will receive an error, but the executable will be running. It doesn't look like they should actually error, so maybe not a concern.

Reply:
i couldn't find a way to have a "non-blocking error".. maybe i should just catch here and warn?  that would show up in the auraed logs.